File: c++/arrays/arrays_1d/array_traversal_2.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cout << "Enter the size of array :";
    cin >> n;
    
    int arr[n];
    
    cout << "Enter the elements of array"<<endl;
    for(int i = 0; i < n; i++){
        cin >> arr[i];
    }

    cout << "Traversal of array :"<< endl;
    for(int i = 0; i < n; i++){
        cout << arr[i]<<" ";
    }

}

File: java/arrays/arrays_1d/ArrayTraversal_2.java
import java.util.Scanner;
class ArrayTraversal_2{
    public static void main(String args[]){
        int n;
        System.out.print("Enter size of array :");

        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();

        int [] arr = new int[n];
        System.out.println("Enter Array Elements : ");
        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
        }

        System.out.println("Traversal of array : ");
        for(int i = 0; i < n; i++){
            System.out.println("arr["+i+"]"+" = "+ arr[i]);
        }

    }
}